nats.go

```diff
@@ -266,8 +269,15 @@ type Options struct {
 
 	// DisconnectedCB sets the disconnected handler that is called
 	// whenever the connection is disconnected.
+	// Will not be called if DisconnectedErrCB was initiated.
+	// DEPRECATED. use DisconnectedErrCB which passes initial err
 	DisconnectedCB ConnHandler
 
+	// DisconnectedErrCB sets the disconnected error handler that is called
+	// whenever the connection is disconnected.
+	// DisconnectedCB will not be called if DisconnectedErrCB was initiated.
```

Comment:
`DisconnectedCB will not be called if DisconnectedErrCB is set`?

Reply:
thanks, fixed docs